packages/react/src/app-layout/AppLayoutHeaderBrand.tsx

```diff
@@ -42,12 +42,17 @@ export function AppLayoutHeaderBrand({
 
 	return logo && secondLogo ? (
 		<Flex
-			css={{ overflow: 'hidden' }}
+			css={{
+				overflow: 'hidden',
+				':focus-within': {
+					overflow: 'visible',
```

Comment:
If we don't need `overflow: hidden` when it has focus, then we don't need it at all? And if we do need it, then wouldn't this unexpectedly change the UI when it has focus?

Reply:
That reminds me of what I forgot to test, the second logo wrapping. We use overflow hidden to hide the vertical divider when the content dynamically wraps.

- [ ] test dynamic wrapping and visibility of vertical divider when second logo is focused

Reply:
I've ended up using the same technique as the single logo version with dispersed padding.